package.json

```
@@ -69,7 +69,7 @@
     "concurrently": "^9.0.1",
     "http-server": "^14.1.1",
     "lodash": "^4.17.21",
-    "rimraf": "^6.0.1",
+    "rimraf -g": "^6.0.1",
```

your find-and-replace was a little too eager here :D
```suggestion
    "rimraf": "^6.0.1",
```